raft/src/main/java/org/apache/kafka/snapshot/Snapshots.java

```diff
@@ -16,16 +16,21 @@
  */
 package org.apache.kafka.snapshot;
 
+import org.apache.kafka.raft.OffsetAndEpoch;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.text.NumberFormat;
 import java.util.Optional;
-import org.apache.kafka.raft.OffsetAndEpoch;
 
 public final class Snapshots {
-    private static final String SUFFIX =  ".checkpoint";
+    private static final Logger log = LoggerFactory.getLogger(Snapshots.class);
+    private static final String SUFFIX = ".checkpoint";
     private static final String PARTIAL_SUFFIX = String.format("%s.part", SUFFIX);
+    private static final String DELETE_SUFFIX = ".deleted";
```

Comment:
@dengziming Sorry that I didn't notice this earlier. Why don't we make the `DELETE_SUFIX` similar to the `PARTIAL_SUFFIX`? E.g. `public static final String DELETE_SUFFIX = String.format("%s.deleted", SUFFIX);`

This will give us the follow filename suffixes:

1. Immutable snapshots - `.checkpoint`
2. Partial snapshots - `.checkpoint.part`
3. Deleted snapshots - `checkpoint.deleted`

Reply:
Sorry I made a mistake when understanding `Log.deleteSegmentFiles`, I will change it.